### ISSUE TITLE
Additional configuration options

### ISSUE THREAD

**Issue by csabia** [optionfactory/gdrive2slack#2]
Notification time intervals
Caching strategy
Name and picture avatar for slack posts

**Comment by mmartain** [comment]
Yes, for example make the title of the slack notification meaningful so we can deduce the file and author from the push notification on my phone.
